modules/options/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -312,11 +312,21 @@ final case class BuildOptions(
     repositories: Seq[Repository] = Nil
   ): Either[BuildException, Option[ScalaParameters]] = either {
 
+    val defaultVersions = Set(
+      Constants.defaultScalaVersion,
+      Constants.defaultScala212Version,
+      Constants.defaultScala213Version
+    )
+
     val svOpt: Option[String] = scalaOptions.scalaVersion match {
       case Some(MaybeScalaVersion(None)) =>
         None
+      // v Do not validate Scala version
```

Comment:
NIT these comments aren't entirely readable to me.
```suggestion
         // do not validate the default Scala version when none has been passed explicitly
        None
```